.circleci/config.yml

```diff
@@ -175,13 +175,22 @@ jobs:
             terraform apply -auto-approve -var-file="$TERRAFORM_ENV_CONFIG_FILE"
 
 workflows:
+  pull-request:
```

Comment:
just a food for thought: we could also use the `filter` on each job to include or exclude certain branches. That way you would have only one workflow. Ex: https://support.circleci.com/hc/en-us/articles/115015953868-Filter-workflows-by-branch-

Reply:
oh I just saw that it was using filters and you removed it :D